src/engraving/rendering/dev/chordlayout.cpp

```diff
@@ -175,7 +175,7 @@ void ChordLayout::layoutPitched(Chord* item, LayoutContext& ctx)
     if (item->arpeggio()) {
         item->arpeggio()->findAndAttachToChords();
         item->arpeggio()->mutldata()->maxChordPad = 0.0;
-        static constexpr int MAX_ARPEGGIO_X = 10000;
+        static constexpr int MAX_ARPEGGIO_X = MAX_INT;
         item->arpeggio()->mutldata()->minChordX = MAX_ARPEGGIO_X;
```

Comment:
```suggestion
        item->arpeggio()->mutldata()->minChordX = DBL_MAX;
```
(DBL_MAX, not INT_MAX, because actually `minChordX` is a double)